testing/testcafe/tests/common/draggable.ts

```diff
@@ -0,0 +1,71 @@
+import { ClientFunction, Selector } from 'testcafe';
+import url from '../../helpers/getPageUrl';
+import createWidget from '../../helpers/createWidget';
+
+fixture.disablePageReloads`Draggable`
+  .page(url(__dirname, '../container.html'));
+
+test('dxDraggable element should not loose its position on dragging with auto-scroll inside ScrollView (T1169590)', async (t) => {
+  const draggable = Selector('#drag-me');
+
+  await t
+    .drag(draggable, 0, 400, { speed: 0.1 })
+
+    .expect(Selector('.dx-scrollable-container')().scrollTop)
+    .gt(95);
+
+  await t
+    .expect((await draggable().boundingClientRect).top)
+    .gt(400);
+
+  await t.scrollIntoView(draggable);
+
+  await t
+    .drag(draggable, 400, 0, { speed: 0.1 })
+
+    .expect(Selector('.dx-scrollable-container')().scrollLeft)
+    .gt(95);
+
+  await t
+    .expect((await draggable().boundingClientRect).left)
+    .gt(400);
+}).before(async () => {
+  const init = ClientFunction(() => {
```

Comment:
Let's move the ```init``` function from the ```before``` block.
E.g. on the tip of the file.

Reply:
There is tslint error in this case: 'init' was used before it was defined.